.github/workflows/updatecli.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pipeline-name: [ beats, golang, hermit ]
+        pipeline-name: [ beats, golang, hermit, mods ]
```

Comment:
Should we also do it on the backport?

Reply:
I think we should avoid backporting dependency updates to keep released or to-be-released versions static to avoid potential issues.